src/main/java/com/pusher/rest/data/Event.java

```diff
@@ -0,0 +1,45 @@
+package com.pusher.rest.data;
+
+import com.pusher.rest.util.Prerequisites;
+
+/**
+ * POJO for JSON encoding of trigger batch events.
+ */
+public class Event {
+
+    private final String channel;
+    private final String name;
+    private final String data;
+    private final String socket_id;
```

Comment:
Always use camel case. The Gson serialiser is configured to translate to snake case in the serialised output.